sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -2053,9 +2053,10 @@ object SQLConf {
     .createWithDefault(false)
 
   val IGNORE_INVALID_PARTITION_PATHS = buildConf("spark.sql.files.ignoreInvalidPartitionPaths")
-    .doc("Whether to ignore invalid partition paths that do not match <column>=<value>. When " +
-      "the option is enabled, table with two partition directories 'table/invalid' and " +
-      "'table/col=1' will only load the latter directory and ignore the invalid partition")
+    .doc("Whether to ignore invalid partition paths that do not match " +
+      "&lt;column&gt;=&lt;value&gt;. When the option is enabled, table with two partition " +
```

Comment:
Oh, this is interesting one. Should we actually try to escape them in `./sql/gen-sql-api-docs.py`?

Reply:
I would like to do that but I was considering it might be tricky to distinguish them from valid HTML tags.

Reply:
Does it work if you do this with `` `<column>=<value>` ``?

Reply:
ok